src/Messages/Baseline4R1/ComplexMessages.cs

```diff
@@ -138,10 +152,11 @@ public record PointPdiVersionCheckMessage (string SenderIdentifier, string Recei
         return new PointPdiVersionCheckMessage(SenderIdentifier, ReceiverIdentifier, ResultPdiVersionCheck, SenderPdiVersion, ChecksumLength, Checksum);
     }
 
-    public override byte[] ToByteArray() {
-        var result = new byte[46 + ChecksumLength];
+    public override byte[] ToByteArray()
+    {
+        var result = new byte[44 + ChecksumLength];
         result[0] = (byte)ProtocolType.Point;
-        BitConverter.GetBytes(0x0025).Take(2).ToArray().CopyTo(result, MessageTypeOffset);
```

Comment:
Why did this change? Should be 0x0025 everywhere.

<img width="577" alt="Screenshot 2023-06-19 at 14 28 33" src="https://github.com/eulynx-live/subsystems/assets/2394156/67678edd-beea-4cb6-945c-04876bc219a0">

Reply:
Not sure why this file was like that, don't remember changing it, but well... I fixed it